src/com/rcv/Tabulator.java

```diff
@@ -68,12 +68,7 @@ static TieBreakMode tieBreakModeForConfigSetting(String setting) {
   private int contestId;
   private List<String> contestOptions;
   private ElectionConfig config;
-
```

Comment:
oops

Reply:
Just looking at the Tabulator file it's getting a bit unwieldy.  How would you feel about moving Tiebreak / Overvote stuff into separate files, only if it's not making things more complicated?

Reply:
Maybe a helper or utility file... not sure, just spitballing here.

Reply:
Tiebreak, at least, could easily be a separate file. I'll send a PR for that.